### ISSUE TITLE
@babel/preset-env does not use .browserslistrc config file at all

### ISSUE THREAD

**Issue by thasmo** [babel/babel#11651]
## Bug Report

**Current behavior**
Using the mentioned babel config it seems that `@babel/preset-env` does not use the `.browserslistrc` config file at all. When defining the `targets` option manually, it works as expected.

**babel.config.js**

```js
'use strict';

module.exports = function (api) {
    api.cache(true);

    return {
        presets: [
            [
                '@babel/preset-env',
                {
                    modules: false,
                    useBuiltIns: 'usage',
                    shippedProposals: true,
                    corejs: {
                        version: 3,
                        proposals: true,
                    },
                },
            ],
        ],
    };
};
```

**Expected behavior**
`@babel/preset-env` should respect the settings defined in the `.browserslistrc` file.

**Debug Output**
```

i Compiling Webpack
@babel/preset-env: `DEBUG` option

Using targets:
{}

Using modules transform: false
...
```

**Environment**

```
  System:
    OS: Windows 10 10.0.18363
  Binaries:
    Node: 12.16.1 - c:\program files\nodejs\node.EXE
    npm: 6.14.4 - c:\program files\nodejs\npm.CMD
  npmPackages:
    @babel/core: ^7.10.1 => 7.10.1
    @babel/plugin-syntax-dynamic-import: ^7.8.3 => 7.8.3
    @babel/preset-env: ^7.10.1 => 7.10.1
    eslint: ^7.1.0 => 7.1.0
    webpack: ^4.43.0 => 4.43.0
```

**Context**
Used with `babel-loader` via `webpack`.

**Comment by existentialism** [comment]
@thasmo can you create a small repo that reproduces the problem?

**Comment by thasmo** [comment]
Hey! I've figured out what's going right at this very moment. Two things in combination really tripped me up.

First, the `babel-loader` cache is not invalidated when the `.browserslistrc` file contents are changed, so my `.browserslistrc` configuration was cached and not updated.

Second, I've used the query `chrome >= 82` in the `.browserslistrc` file exclusively to test this, but somehow `browserslist` has some weird browser version validation and deems `Chrome 83` to be an unknown browser/version, removes it from the configuration and back comes an empty configuration. (Although Chrome 83 is the latest stable release version already.)

These two things just wasted hours of my time but I'm glad I found these issues.
